ravenframework/SupervisedLearning/SupervisedLearning.py

```diff
@@ -359,7 +364,11 @@ def evaluate(self,edict):
           featureValues[:, :, cnt] = ((values[names.index(feat)] - self.muAndSigmaFeatures[feat][0]))/self.muAndSigmaFeatures[feat][1]
         else:
           featureValues[:,cnt] = ((values[names.index(feat)] - self.muAndSigmaFeatures[feat][0]))/self.muAndSigmaFeatures[feat][1]
-    return self.__evaluateLocal__(featureValues)
+    target = self.__evaluateLocal__(featureValues)
+    # if self.target[0] in self.muAndSigmaFeatures.keys():
+    if ('normalizeTargets' in self.info.keys()) and self.info['normalizeTargets']:
+      target.update((x, y * self.muAndSigmaTargets[self.target[0]][1] + self.muAndSigmaTargets[self.target[0]][0]) for x, y in target.items())
```

Comment:
Similar comment here, you are only using the first target to do the normalization, I think we need to extend it.